blivet/flags.py

```diff
@@ -109,7 +108,6 @@ def update_from_boot_cmdline(self):
             self.noiswmd = True
 
     def update_from_anaconda_flags(self, anaconda_flags):
-        self.installer_mode = True
```

Comment:
I agree that yes, eventually it should. For now I will move it to blivet.osinstall with all the other installer-specific items temporarily living there.